scripts/toc.js

```diff
@@ -12,11 +12,23 @@ const yargs = require("yargs");
 const yargv = yargs
   .strict()
   .usage("Usage: ./scripts/toc.js <notes-folder>")
-  .example("./scripts/toc.js es9/2018-03/");
+  .example("./scripts/toc.js es9/2018-03/")
+  .option("all", {
+    alias: "a",
+    default: false
+  });
 
 const argv = yargv.argv;
 
-if (argv._.length) {
+if (argv.all) {
+  glob('./es+([0-9])/20[1-2][0-9]-[0-3][0-9]', (error, results) => {
```

Comment:
we would need to fix this before 2030

Reply:
You'll have 9 years to fix it after 2030!